mmv1/templates/terraform/resource_iam.html.markdown.erb

```diff
@@ -202,7 +202,7 @@ The following arguments are supported:
 * `<%= param.name.underscore -%>` - (Required) <%= param.description -%> Used to find the parent resource to bind the IAM policy to
 <% end -%>
 <% end -%>
-<% if object.base_url.include?("{{project}}")-%>
+<% if object.iam_policy.base_url.include?("{{project}}")-%>
```

Comment:
I think we need a nil check on the iam policy's `base_url`. This is failing to build probably because the IAM policy does not always have its own base_url, and in most cases inherits from the parent object